rust/foxglove/src/mcap_writer.rs

```diff
@@ -43,6 +43,7 @@ impl McapWriter {
     }
 
     /// Instantiates a new MCAP writer with the provided options.
+    /// The library option is ignored. chunk_size=0 is treated
```

Comment:
incomplete comment

Also, is ignoring the library option a bug? Should we not expose an option if it is ignored?

Reply:
It's not our type, it's from the mcap library. That's an argument maybe for duplicating the type and providing a conversion instead.